### PR TITLE
Update README.md to add required step for local development

### PR DESCRIPTION
**Pull request by Nabeelshar**
add required step to run hoppscotch on local device

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,9 +292,10 @@ _Sample keys only work with the [production build](https://hoppscotch.io)._
 
 1. [Clone this repo](https://help.github.com/en/articles/cloning-a-repository) with git.
 2. Install pnpm using npm by running `npm install -g pnpm`.
-3. Install dependencies by running `pnpm install` within the directory that you cloned (probably `hoppscotch`).
-4. Start the development server with `pnpm run dev`.
-5. Open the development site by going to [`http://localhost:3000`](http://localhost:3000) in your browser.
+3. Run `cp .env.example .env`(update them with your own info)
```
Comment (AndrewBastin) [inline README.md:295]: Isn't this already mentioned in Step 0 (Developing section).

Comment (Nabeelshar) [reply README.md:295]: There is difference step 0 is suggesting for production build, while sample keys work with local build.

Comment (AndrewBastin) [reply README.md:295]: That is an intentional decision. Although we do allow the keys to work by default with local build, we don't recommend building with it as if you make a misconfiguration or something you can end up with messed up user data in the production firebase.

**Comment by AndrewBastin** [comment]
Closing this PR as this was an intended statement.

**Pull request closed without merging**